lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
This looks great! But it isn't doing anything at the moment. There's one last thing you need to add for it to actually get used. Is there a particular file contained in tarballs that is indicative of the Bazel build system? You'll have to add that file to the `clues` list in the `BuildSystemGuesser` class below.